oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/IndexConstants.java

```diff
@@ -27,6 +27,11 @@ public interface IndexConstants {
 
     String TYPE_PROPERTY_NAME = "type";
 
+    /**
+     * Property to hold the value for original index implementation in case the index type is set to disabled.
+     */
+    String ORIGINAL_TYPE_PROPERTY_NAME = "orig_type";
```

Comment:
two questions:
* should we use a hidden property here prefixing it with `:`?
* I would use the camel case notation to be consistent with the other constants (eg: `originalType` or `origType`)

Reply:
Yes - hidden property might make more sense